cantabular-import/analysis/full-import-export/main.go

```diff
@@ -239,7 +239,7 @@ func main() {
 
 	// then check that state is : 'edition-confirmed' ... under some sort of repeat timeout
 
-	attempts := 50
+	attempts := 200
 
 	for attempts > 0 {
 		time.Sleep(100 * time.Millisecond)
```

Comment:
maybe we can create a contant `StateVerifyPeriod=100 * time.Millisecond` at the beginning of the file, then do `time.Sleep(StateVerifyPeriod)`